nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/WEB-INF/partials/canvas/settings-content.jsp

```diff
@@ -23,7 +23,7 @@
             <div class="clear"></div>
         </div>
         <div id="settings-tabs-content">
-            <button id="new-service-or-task" class="add-button fa fa-plus" title="Create a new reporting task controller service" style="display: block;"></button>
+            <button id="new-service-or-task" class="add-button fa fa-plus" title="Create a new reporting task/flow analysis rule controller service" style="display: block;"></button>
```

Comment:
While not introduced here, I wonder if we should update the title here. Thoughts on just changing this to `Create New`?

Reply:
I checked and the title of that button is overridden for each controller-level component.
If not it's supposed to be a bug.

I think we could remove the title attribute here.
Or maybe add something that would alert us when it hasn't been overridden. 
Or I can simply revert this change.